assets/css/styles.css

```diff
@@ -110,24 +122,45 @@ body  {
 }
 
 .hero__btn {
-    background-color: #ffcc51;
-    text-transform: uppercase;
-    color: #1c7aa7;
+    line-height: 45px;
+    height: 45px;
+    width: 75%;
+    background-color: #1c7aa7;
     font-weight: bold;
+    text-align: center;
+    border: #1c7aa7 1px solid;
+    border-radius: 5px;
     position: absolute;
     top: 90%;
     left: 50%;
     transform: translateX(-50%) translateY(-50%);
 }
 
-h2 {
-    margin-bottom: 60px;
-    font-size: 50px;
-    font-style: oblique;
+.hero__btn--text {
+     text-transform: uppercase;
+     color: #fff;
+     font-size: 1.4em;
+     letter-spacing: 0.3em;
+     padding: 0 50px;
+ }
+
+.hero__btn:hover {
+    background-color: #f1e4b0;
+}
+
+
+.hero__btn--text:hover {
```

Comment:
Please decide weather you are using BEM or other kind of naming convention. Whatever works, but stick to one in the entire project.

Reply:
We've decided to use BEM and everyone will fix code in their part of the project. I haven't found any mistakes in BEM convention in my sections (hero and team) but I'm not sure of course. Could you confirm I'm right please.